src/main/scala/codacy/plugins/test/JsonTests.scala

```diff
@@ -1,40 +1,40 @@
 package codacy.plugins.test
 
-import java.nio.file.Path
-
-import codacy.utils.{CollectionHelper, Printer}
+import codacy.utils.CollectionHelper
 import com.codacy.plugins.api.PatternDescription
 import com.codacy.plugins.api.results.Pattern
 import com.codacy.plugins.results.traits.DockerToolDocumentation
 import com.codacy.plugins.utils.BinaryDockerHelper
+import better.files._
+import java.io.{File => JFile}
 
 object JsonTests extends ITest {
 
   val opt = "json"
 
-  def run(testSources: Seq[Path], dockerImage: DockerImage, optArgs: Seq[String]): Boolean = {
-    Printer.green("Running JsonTests:")
+  def run(docsDirectory: JFile, dockerImage: DockerImage, optArgs: Seq[String]): Boolean = {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method is longer than 50 lines](https://app.codacy.com/app/Codacy/codacy-plugins-test/pullRequest?prid=4500290)